applications/CompressiblePotentialFlowApplication/custom_elements/transonic_perturbation_potential_flow_element.cpp

```diff
@@ -996,11 +997,11 @@ void TransonicPerturbationPotentialFlowElement<TDim, TNumNodes>::AssignRightHand
     {
         rRightHandSideVector[rRow] = rUpper_rhs(rRow);
         // check if should be zero
-        rRightHandSideVector[rRow + TNumNodes] = rWake_rhs(rRow);
+        rRightHandSideVector[rRow + TNumNodes] = 0.0; // rWake_rhs(rRow);
```

Comment:
I think this was already changed in the last PR, maybe try merging with master to get the updated version